plugins/generator-1.18.2/forge-1.18.2/aitasks/follow_entity.java.ftl

```diff
@@ -1,2 +1,4 @@
 <#include "aiconditions.java.ftl">
-this.goalSelector.addGoal(${customBlockIndex+1}, new FollowMobGoal(this, (float)${field$speed}, ${field$maxrange}, ${field$followarea})<@conditionCode field$condition/>);
\ No newline at end of file
+<#if !data.waterMob || data.flyingMob>

```

Comment:
Is this the right condition? Isn't this the same effect as using 

`<#if !data.waterMob>` ?

Or does AI work with flying + water mobs?

Just asking as other AI tasks use

`<#if !data.flyingMob && !data.waterMob>`

Reply:
It's because the flying mob option has the priority over the water mob option (see #3092, where the same condition was used)